### PR TITLE
Liquid cooling

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

File: blog/content/post/2023-10-22-liquid-cooling.md
Comment (pietrushnic) [inline blog/content/post/2023-10-22-liquid-cooling.md:43]: @miczyg1 if you upgrade on top of some standard products, which can be found in the 3mdeb shop, it would be worth mentioning otherwise, please mention more information about the spec e.g. chassis model.

File: blog/content/post/2023-10-22-liquid-cooling.md
Comment (pietrushnic) [inline blog/content/post/2023-10-22-liquid-cooling.md:70]: I would say the embed version would be better.

File: blog/content/post/2023-10-22-liquid-cooling.md
Comment (pietrushnic) [inline blog/content/post/2023-10-22-liquid-cooling.md:122]: Can we point to Dasharo on MSI and fan control?

File: blog/content/post/2023-10-22-liquid-cooling.md
Comment (pietrushnic) [reply blog/content/post/2023-10-22-liquid-cooling.md:43]: There is no word this is Dasharo-based hardware, which IMHO is very important; maybe even mention that in the abstract and title.

File: blog/content/post/2023-10-22-liquid-cooling.md
Comment (pietrushnic) [inline blog/content/post/2023-10-22-liquid-cooling.md:147]: I would need two things here:
- promotion of your Twitter account - if you like content like this feel free to flow me yada yada
- promotion of 3mdeb account in similar fashion

**Review by pietrushnic — CHANGES_REQUESTED** [review]
Overall post is very cool ;)
What is needed are some cosmetic improvements and Grammarly checks.

File: blog/content/post/2023-10-22-liquid-cooling.md
Comment (mkopec) [inline blog/content/post/2023-10-22-liquid-cooling.md:121]: Have the pump speed follow CPU temp, and fan speed follow liquid temperature (if you have a temperature probe in there). This yields best results for me - quiet at idle, but cool under load :)

**Comment by pietrushnic** [comment]
@miczyg1 it would be great if you could access the review. This is very close to being published.

File: blog/content/post/2023-10-22-liquid-cooling.md
Comment (miczyg1) [reply blog/content/post/2023-10-22-liquid-cooling.md:121]: Yeah gladly, but the pump does not have PWM pin (3pins only) so cannot be controlled. Maybe it could be controlled by voltage, but I am not sure if the pump will like undervoltage.

File: blog/content/post/2023-10-22-liquid-cooling.md
Comment (miczyg1) [reply blog/content/post/2023-10-22-liquid-cooling.md:130]: We have no release yet which have the fan control, so I can not put an URL. But I have mentioned that it is coming in v1.1.3

File: blog/content/post/2023-10-22-liquid-cooling.md
Comment (miczyg1) [reply blog/content/post/2023-10-22-liquid-cooling.md]: https://github.com/3mdeb/news-and-ideas/pull/545/commits/2b5b9cc842c5f672f7fc60bacf4139b5fa6b19d6#diff-bd4456b5082aa26d9ea51ebc4ce2ed44da6fd4f97d01097d5221951114a52590R43-R46

File: blog/content/post/2023-10-22-liquid-cooling.md
Comment (miczyg1) [reply blog/content/post/2023-10-22-liquid-cooling.md:76]: https://github.com/3mdeb/news-and-ideas/pull/545/commits/2b5b9cc842c5f672f7fc60bacf4139b5fa6b19d6#diff-bd4456b5082aa26d9ea51ebc4ce2ed44da6fd4f97d01097d5221951114a52590L69-R79

File: blog/content/post/2023-10-22-liquid-cooling.md
Comment (miczyg1) [reply blog/content/post/2023-10-22-liquid-cooling.md:130]: https://github.com/3mdeb/news-and-ideas/pull/545/commits/2b5b9cc842c5f672f7fc60bacf4139b5fa6b19d6#diff-bd4456b5082aa26d9ea51ebc4ce2ed44da6fd4f97d01097d5221951114a52590R131-R134

File: blog/content/post/2023-10-22-liquid-cooling.md
Comment (miczyg1) [reply blog/content/post/2023-10-22-liquid-cooling.md:171]: https://github.com/3mdeb/news-and-ideas/pull/545/commits/2b5b9cc842c5f672f7fc60bacf4139b5fa6b19d6#diff-bd4456b5082aa26d9ea51ebc4ce2ed44da6fd4f97d01097d5221951114a52590R151-R162

**Comment by miczyg1** [comment]
> @miczyg1 it would be great if you could access the review. This is very close to being published.

Sorry, I forgot about it a bit recently. All discussion should be addressed @pietrushnic

**Review by pietrushnic — APPROVED** [review]

**Comment by pietrushnic** [comment]
@mkopec, it looks like you resolved but didn't accept the changes; feel free to do so if you have no additional comments. @miczyg1 I approved and I think this is ready for publishing as soon as we will get approval from @mkopec

**Comment by mkopec** [comment]
Oh I resolved it because I didn't want it to block merging. Approved @pietrushnic @miczyg1

**Review by mkopec — APPROVED** [review]

**Pull request merged**
